app/assets/javascripts/admin/addon/components/site-settings/bool.gjs

```diff
@@ -0,0 +1,39 @@
+import Component from "@glimmer/component";
+import { on } from "@ember/modifier";
+import { action, computed } from "@ember/object";
+import { htmlSafe } from "@ember/template";
+import { isEmpty } from "@ember/utils";
+
+export default class Bool extends Component {
+  @computed("value")
+  get enabled() {
+    if (isEmpty(this.args.value)) {
+      return false;
+    }
+    return this.args.value.toString() === "true";
+  }
+
+  set enabled(value) {
+    this.args.changeValueCallback(value ? "true" : "false");
+  }
+
+  @action
+  onToggle(event) {
+    if (event.target.checked) {
+      this.enabled = true;
```

Comment:
enabled should be a tracked property no?

Reply:
No, it's currently a setter which calls `changeValueCallback`, but I will just remove this indirection and call `changeValueCallback` directly